### ISSUE TITLE
Validator says; Text run is not in Unicode Normalization Form C

### ISSUE THREAD

**Issue by tarquinwj** [validator/validator#1740]
```
<!DOCTYPE html>
<title>Test</title>
<p>&#8053;</p>
```

This gives the following warning:
Warning: Text run is not in Unicode Normalization Form C.

The same warning happens when using the hex notation &#x1F75; or when using separate base character and diacritic &#x03B7;&#x0301;

However, the codepoints on either side do not get this warning: &#8052; &#8054;

I do not understand this warning well enough to know if this is an error in the validator, but it seems wrong, especially since x1F73 is the same base character with a grave accent instead of an acute accent, and that is allowed. All browser engines I tested seem to understand it no matter which notation I use.

**Comment by sideshowbarker** [comment]
So, according to `com.ibm.icu.text.Normalizer.normalize` https://unicode-org.github.io/icu-docs/apidoc/dev/icu4j/com/ibm/icu/text/Normalizer.html#normalize(char%5B%5D,char%5B%5D,com.ibm.icu.text.Normalizer.Mode,int), you’re apparently supposed to instead use `ή` (`U+03AE` or `&#x03AE` or `&#942`).

But I don’t know how you or I are expected to know that, or to ever figure it out on our own.

So, I just now pushed a change to the checker which in cases like this causes it to also emit the correct “normalized” text:

> Text run is not in Unicode Normalization Form C. Should instead be “ή”. (Copy and paste that into your source document to replace the un-normalized text.)
